comic.js

```diff
@@ -1 +1,6 @@
-document.body.innerHTML += '<style>* { font-family: "Comic Sans MS";} p:nth-child(odd) {color: pink;} p:nth-child(even) {color: red;} li:nth-child(even) {color: orange;}</style>';
\ No newline at end of file
+document.body.innerHTML += 
```

Comment:
Better but not sure yet. Maybe we should set up a committee to discuss what standard to use for linebreak formatting.

Reply:
WHATEVER

Reply:
/ban @Kaivosukeltaja

Reply:
lgtm